scripts/netlify.js

```diff
@@ -0,0 +1,10 @@
+const shell = require('shelljs');
+
+shell.exec('git clone https://github.com/videojs/http-streaming');
+shell.cd('http-streaming');
+shell.exec('npm ci');
+shell.exec('npm link ../');
+shell.exec('npm run netlify');
+shell.cp('-R', 'deploy',  '../');
```

Comment:
if we wanted to, we could configure it to deploy `http-streaming/deploy` then we don't need these 3 lines.

Reply:
We could, but I think it's better to run these lines and keep netlify `deploy` in a predictable place for all of our projects.